arm-software/embedded/llvmlibc-support/crt0/memory_a.h

```diff
@@ -85,8 +85,9 @@ void setup() {
   // Put the page table in the .init section so it doesn't later get
   // zero-initialized.
   static_assert(sizeof(unsigned long) == PAGE_TABLE_ENTRY_SIZE);
-  static volatile unsigned long pagetable[PAGE_TABLE_ENTRY_COUNT]
-      __attribute__((section(".init"), aligned(PAGE_TABLE_ALIGNMENT)));
+  [[gnu::section(".init"),
+    gnu::aligned(PAGE_TABLE_ALIGNMENT)]] static volatile unsigned long
+      pagetable[PAGE_TABLE_ENTRY_COUNT];
```

Comment:
```suggestion
  [[gnu::section(".init"), gnu::aligned(PAGE_TABLE_ALIGNMENT)]]
  static volatile unsigned long pagetable[PAGE_TABLE_ENTRY_COUNT];
```

Reply:
(Optional formatting nit: this is one line shorter _and_ seems more legible to me. But don't worry about it if you can't persuade clang-format to see it this way!)